src/main/python/apache/aurora/executor/common/sandbox.py

```diff
@@ -182,6 +182,10 @@ def _create_symlinks(self):
     try:
       safe_mkdir(mesos_host_sandbox_root)
       os.symlink(os.environ['MESOS_SANDBOX'], self._mesos_dir)
+      # Restore permissions to pre Mesos 1.6 levels in order to retain Aurora task ssh
```

Comment:
Please mark this as FIXME to better indicate that we want a cleaner solution. Also please add link to the Mesos issue (as done here https://github.com/apache/aurora/pull/42/files#diff-bb9116a02fccc1c8feb7d6c797f52384R141)

Reply:
Done, thanks for the review!